.github/workflows/manifold.yml

```diff
@@ -21,25 +21,54 @@ jobs:
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    - uses: actions/checkout@v2
-    - uses: jwlawson/actions-setup-cmake@v1.12
-    
     - name: Install dependencies
       run: |
         apt-get -y update
-        DEBIAN_FRONTEND=noninteractive apt install -y libglm-dev libassimp-dev libcgal-dev
-    - name: Build CUDA
+        DEBIAN_FRONTEND=noninteractive apt install -y libglm-dev libassimp-dev libcgal-dev libomp-dev nodejs git python3
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - uses: jwlawson/actions-setup-cmake@v1.12
+    - name: Build WASM
       run: |
-        mkdir buildCUDA
-        cd buildCUDA
-        cmake -DCMAKE_BUILD_TYPE=Release .. && make
+        patch third_party/assimp/code/CMakeLists.txt < assimp.diff
+        mkdir buildWASM
+        cd buildWASM
+        git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
Should this just be another git submodule for consistency?

Reply:
This is just for setting up the build environment, not build dependency. This is the official instruction for installing emscripten on Ubuntu, but other distros have their own packages and don't have to do this.